generator/config/search/geoShape.yaml

```diff
@@ -0,0 +1,171 @@
+# $schema: ../schema.json
+name: geoShape
+link: 'https://www.mongodb.com/docs/atlas/atlas-search/geoShape/'
+type:
+    - searchOperator
+encode: object
+description: |
+    The geoShape operator supports querying shapes with a relation to a given 
+    geometry if indexShapes is set to true in the index definition.
+arguments:
+    -
+        name: path
+        type:
+            - searchPath
+    -
+        name: relation
+        type:
+            - string # contains | disjoint | intersects | within
+    -
+        name: geometry
+        type:
+            - geometry
+    -
+        name: score
+        optional: true
+        type:
+            - searchScore
+tests:
+    -
+        name: 'Disjoint'
+        link: 'https://www.mongodb.com/docs/atlas/atlas-search/geoShape/#disjoint-example'
+        pipeline:
+            -
+                $search:
+                    geoShape:
+                        relation: 'disjoint'
+                        geometry:
+                            type: 'Polygon'
+                            coordinates:
+                                -
+                                    -
+                                        - -161.323242
+                                        - 22.512557
```

Comment:
Would it be more readable to inline these, like `[ -161.323242, 22.512557 ]`? That might reduce a lot of lines in this file.

If there's a concern that it's generated and might conflict with future syncs, no worries leaving as-is. I'm not familiar with how these are imported.

Reply:
It's generated by my `js2yaml.html` file. I've made a few editions in some places, so why not...